src/services/package-manager/package-service.ts

```diff
@@ -266,38 +203,40 @@ class PackageService {
 
     private async exportPackagesAndAssets(nodes: BatchExportNodeTransport[]): Promise<any[]> {
         const zips = [];
-        const packages = nodes as ContentNodeTransport[];
-        for (const rootPackage of packages) {
-            const exportedPackage = await packageApi.exportPackage(rootPackage.key)
+        for (const rootPackage of nodes) {
+            const exportedPackage = await packageApi.exportPackage(rootPackage.key, rootPackage.version.version)
             zips.push({
                 data: exportedPackage,
-                packageKey: rootPackage.key
+                packageKey: rootPackage.key,
+                version: rootPackage.version.version
             });
         }
         return zips;
     }
 
-    private async exportToZip(nodes: BatchExportNodeTransport[]): Promise<void> {
-        const manifestNodes = this.exportManifestOfPackages(nodes);
+    private async exportToZip(nodes: BatchExportNodeTransport[], versionsByNodeKey: Map<string, string[]>): Promise<void> {
+        const manifestNodes = this.exportManifestOfPackages(nodes, versionsByNodeKey);
         const packageZips = await this.exportPackagesAndAssets(nodes);
 
         const zip = new AdmZip();
 
         zip.addFile("manifest.yml", Buffer.from(YAML.stringify(manifestNodes), "utf8"));
         for (const packageZip of packageZips) {
-            zip.addFile(packageZip.packageKey + ".zip", packageZip.data)
+            zip.addFile(`${packageZip.packageKey}_${packageZip.version}.zip`, packageZip.data)
         }
-        zip.writeZip("export_" + uuidv4() + ".zip");
-        logger.info("Successfully exported package");
+
+        const fileName = "export_" + uuidv4() + ".zip";
+        zip.writeZip(fileName);
+        logger.info("Successfully exported package to: " + fileName);
```

Comment:
To stick with the structure, how would it be to use "File downloaded successfully. New filename: " this message on output?